source/docs/appendices/roadmap.md

```diff
@@ -0,0 +1,73 @@
+---
+id: roadmap
+title: Roadmap
+---
+
+Front-Commerce is built around the motto "Make it work, then make it better". This means that it is possible **today** to develop and deploy your website to production. But we still have a lot of things to work on to accomplish our mission: make your life easier, and your customers happier.
+
+So let's have a look at what is coming in Front-Commerce.
+
+<blockquote class="note">
+But keep in mind that our main strength is our flexibility. We do take into account our clients needs and adapt our Roadmap to match theirs. If you have any particular needs, feel free to [contact us](mailto:contact@front-commerce.com) and we will see what we can do to help you :)
+</blockquote>
+
```

Comment:
Do you think we should add here a paragraph about our release process and our wish to migrate existing project during `rc` versions to ensure the migration path is seamless?